packages/devtools_app/lib/src/screens/memory/framework/memory_controller.dart

```diff
@@ -81,7 +78,7 @@ class MemoryController extends DisposableController
 
   static const _jsonKey = 'data';
 
-  Future<void> _init(
+  void _init(
```

Comment:
I'm not concerned about this in this particular scenario, but out of curiosity, is this generally considered best practice? I only ask because since this is an `async` method, it's possible that asynchronous gaps can be introduced that impact execution ordering in some contexts that could lead to some nasty race conditions, especially if it's not clear that asynchronous work is being done. Having to wrap unawaited asynchronous functions `unawaited` makes it clear that asynchronous work is being done from the call site.

Reply:
Oh ho! There is a linter rule. https://dart.dev/tools/linter-rules/avoid_void_async

I had no idea. I will drop this :D